ansible/Vagrantfile

```diff
@@ -1,6 +1,6 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "bento/ubuntu-18.04"
-  config.vm.network "forwarded_port", guest: 8080, host: 8080
+  config.vm.network "forwarded_port", guest: 80, host: 9000
```

Comment:
Shouldn't this take 80 on the host and pass it 80 on the guest?

Reply:
The "guest" here maps to the EC2 instance in production, so you want to run the web services on port 80 there.

The "host" here is just your development/testing machine--you don't want to force a root-access port to map to port 80 from guest, so we use a high-numbered one instead (and I changed it to 9000 to avoid the fact that every Vue/Webpack project on the planet wants to serve on ports starting with 8080).

Reply:
Ah, of course